client/header/activity-panel/tab/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import classnames from 'classnames';
+
+export const Tab = ( {
+	icon,
+	title,
+	name,
+	unread,
+	selected,
+	isPanelOpen,
+	onTabClick,
+	index,
+} ) => {
+	const className = classnames( 'woocommerce-layout__activity-panel-tab', {
+		'is-active': isPanelOpen && selected,
+		'has-unread': unread,
+	} );
+
+	let tabIndex = -1;
+
+	// Only make this item tabbable if it is the currently selected item, or the panel is closed and the item is the first item.
+	if ( selected || ( ! isPanelOpen && index === 0 ) ) {
+		tabIndex = null;
+	}
+
+	return (
+		<Button
+			role="tab"
+			className={ className }
+			tabIndex={ tabIndex }
+			aria-selected={ selected }
+			aria-controls={ `activity-panel-${ name }` }
+			key={ `activity-panel-tab-${ name }` }
```

Comment:
Minor, but since this same string is used 2x here should we just make a `const` for it?

Reply:
Fixed, thanks!